packages/utils/src/createSchemaUtils.ts

```diff
@@ -57,7 +60,11 @@ class SchemaUtils<T = any, S extends StrictRJSFSchema = RJSFSchema, F extends Fo
     experimental_defaultFormStateBehavior: Experimental_DefaultFormStateBehavior,
     experimental_customMergeAllOf?: Experimental_CustomMergeAllOf<S>,
   ) {
-    this.rootSchema = rootSchema;
+    if (rootSchema[SCHEMA_KEY] === JSON_SCHEMA_DRAFT_2020_12) {
```

Comment:
So we only want to do this for 2020? What about 2019?

Reply:
Support for embedded schemas has been added to JSONSchema only from draft-2020-12 (see [here](https://json-schema.org/draft/2020-12/release-notes#embedded-schemas-and-bundling)). Therefore, I did not implement support for embedded schemas in prior versions of the standard.
If you want to support this too, it would be a non-standard backport of the feature to previous versions.
As you prefer.